dotnet/samples/AgentSyntaxExamples/AgentInventory.ChatAgents.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Agents;
+using Microsoft.SemanticKernel.Connectors.OpenAI;
+
+namespace AgentSyntaxExamples;
+
+public static partial class AgentInventory
```

Comment:
Why use a partial class?

The classes are both very small and to read the code I need to see content from both together to fully understand how the agents will behave.